File: SoftDev.Tests/CounterTest.cs
using NUnit.Framework;
using SoftDev;

namespace SoftDev.Tests
{
    public class CounterTest
    {
        private Counter counter;

        [SetUp]
        public void Setup()
        {
            counter = new Counter();
        }

        [Test]
        public void GetValue()
        {
            int v = counter.GetValue();
            Assert.AreEqual(0, v);
        }
    }
}

File: SoftDev.Tests/CounterTest.vb
Imports NUnit.Framework
Imports SoftDev.SoftDev

Namespace SoftDev.test

    Public Class CounterTest
        Dim counter As counter

        <SetUp>
        Public Sub Setup()
            Dim counter = New Counter()
        End Sub

        <Test>
        Public Sub GetValue()
            On Error Resume Next
            Dim v As Integer = counter.GetValue()
            Assert.AreEqual(0, v)
        End Sub
    End Class
End Namespace
